app/ts/components/TokenAdd.tsx

```diff
@@ -205,16 +207,24 @@ const QueryStatus = () => {
 				</div>
 			)
 		case 'resolved':
-			const token = serialize(ERC20Token, tokenQuery.value.value)
+			// queried token will have included a checksum address
+			const fetchedToken = tokenQuery.value.value
+			const showChecksumWarning = useComputed(() => {
+				const userInputAddress = noHexPrefix(address.value)
```

Comment:
why do you remove the hex prefix? this also fails to remove `0X`

Reply:
it's now using ethersjs checksum validation which should already do this actually